packages/terra-form-select/src/shared/_Option.jsx

```diff
@@ -105,21 +112,20 @@ const Option = ({
 
   let role = 'option'; // Used for JAWs and VoiceOver on iOS
 
-  /**
-   * VoiceOver in Safari on desktop has issues with role="option" with the combination of the
-   * aria-live section we have and will stutter when reading options.
-   * Switching to role="radio" and role="checkbox" mitigates this behavior.
-   */
-  if (SharedUtil.isSafari() && !('ontouchstart' in window)) {
-    role = 'radio';
+  let itemLabel = display;
+  if (index === 1 || isSelected) {
+    itemLabel = `${expandedStateText} ${display} `;
+    if (!isSelected && SharedUtil.isMac() && SharedUtil.isSafari() && variant !== 'tag' && variant !== 'multiple') {
+      itemLabel += intl.formatMessage({ id: 'Terra.form.select.notselected' });
+    }
```

Comment:
This does not seem to be the proper way to internationalize this string.  You should have a "not selected" message with a placeholder for the display.  You would pass the display as a value to the formatMessage method.  Then the message can be altered by the globalization team as necessary for the language.  We should not just concatenate.

Reply:
Removed concatenate and passed the display as a value to the formatMessage method.